.github/workflows/reviewdog.yml

```
@@ -1,5 +1,5 @@
 name: reviewdog
-on: [pull_request]
+on: [abracadabra]
```

actionlint (1.7.12)

[error] unknown Webhook event "abracadabra". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)